frontend/test/metabase/scenarios/question/reproductions/17910-revision-history-update.js

```diff
@@ -0,0 +1,33 @@
+import { restore, openOrdersTable, modal } from "__support__/e2e/cypress";
+
+export function issue17910() {
+  describe("issue 17910", () => {
+    beforeEach(() => {
+      restore();
+      cy.signInAsNormalUser();
+    });
+
+    it("revisions should work after creating a question without reloading", () => {
```

Comment:
```suggestion
    it("revisions should work after creating a question without reloading (metabase#17910)", () => {
```
It's important to add these kind of references to the test title.